### PR TITLE
Add rosencrantz framework

### LINKED ISSUE #335: Introduce rosencrantz
+ Language : Nim
+ Repo : https://github.com/andreaferretti/rosencrantz
+ Author : @andreaferretti

### PR DESCRIPTION
**Pull request by Willyboar**
After I played a little with rosencrantz framework I thought it would be nice to add it. 

I hope don't mind @andreaferretti.

I checked all the routes with Postman and worked fine. 
 
--------
Closes #335

### CODE REVIEW

File: nim/rosencrantz/server.nim [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  ]
+] ~ post[
+  path("/user")[
+    ok("")
+  ]
+]
+
+let server = newAsyncHttpServer()
+
+waitFor server.serve(Port(8080), handler)
```
Comment (waghanza) [inline nim/rosencrantz/server.nim:24]: Could you change the port to `3000`

File: nim/rosencrantz/server.nimble [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+version = "0.1.0"
+author = "Willyboar"
+description = "Rosencrantz Implementation"
+license = "MIT"
+
+# Dependencies
+
+requires "nim >= 1.0.0"
+requires "rosencrantz >= 0.4.3"
```
Comment (waghanza) [inline nim/rosencrantz/server.nimble:11]: Could you use a range, `>= 0.4.3 & < 0.5`

**Comment by waghanza** [comment]
You also need to add `rosencrantz` to `.travis.yml`

File: nim/rosencrantz/server.nim (hunk 1 above)
Comment (Willyboar) [reply nim/rosencrantz/server.nim]: Done!

File: nim/rosencrantz/server.nimble (hunk 2 above)
Comment (Willyboar) [reply nim/rosencrantz/server.nimble]: Done!

**Comment by Willyboar** [comment]
I think it is ok now.

**Pull request merged**